modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -3598,6 +3598,17 @@
 		"issueKey": "LPD-48759",
 		"to": "addOrUpdateCPAttachmentFileEntry(param#0#, param#1#, param#2#, param#3#, param#4#, param#5#, param#6#, param#7#, param#8#, param#9#, param#10#, param#11#, param#12#, param#13#, param#14#, param#15#, param#16#, param#17#, param#18#, param#19#, null, param#20#, param#21#, param#22#, param#23#, param#24#)"
 	},
+	{
+		"classNames": [
+			"CommerceOrderConstants"
+		],
```

Comment:
@mxs2 hi, since its a static call, the className its direct in the call, wont be a variable so the from can be directly:
`"from": "CommerceOrderConstants.getPaymentStatusLabel(int paramName)",` and remove this `className` attribute